samples/step-by-step/sample.md

```diff
@@ -7,100 +7,191 @@ redirects:
 - nservicebus/nservicebus-step-by-step-guide
 ---
 
-In this sample shows a very simple ordering system that
+This sample illustrates most of the basic NServiceBus concepts in a step-by-step fashion. Download the code from the link above and run it, or build it  following the instructions here.
```

Comment:
> This sample illustrates most of the basic NServiceBus concepts in a step-by-step fashion. 

Why most of? Why not just:
This sample illustrates the basic NServiceBus concepts in a step-by-step fashion.

>Download the code from the link above and run it, or build it  following the instructions here.

Do we need to say run it, build it etc?
How about:
To download the code for this sample click the `Download` link. ( I honestly think that this is overkill also, after all there is a Download button)